src/build/supportedFolders.js

```diff
@@ -6,7 +6,7 @@ exports.extensions = {
     { icon: 'dist', extensions: ['dist', 'out', 'export', 'build'] },
     { icon: 'docs', extensions: ['docs'] },
     { icon: 'flow', extensions: ['flow'] },
-    { icon: 'git', extensions: ['git', 'github'], dot: true },
+    { icon: 'git', extensions: ['github', 'git'], dot: true },
```

Comment:
Sorry, I missed that part. So it's related to the example generator. That makes sense.